example/main-count-fives.cpp

```diff
@@ -109,13 +109,13 @@ using messages_logger=cadmium::logger::logger<cadmium::logger::logger_message_ro
 
 //LOG ALL TO COUT
 using namespace cadmium::logger;
-using info=logger<logger_info, verbatim_formatter, cout_sink_provider>;
-using debug=logger<logger_debug, verbatim_formatter, cout_sink_provider>;
-using state=logger<logger_state, verbatim_formatter, cout_sink_provider>;
-using log_messages=logger<logger_messages, verbatim_formatter, cout_sink_provider>;
-using routing=logger<logger_message_routing, verbatim_formatter, cout_sink_provider>;
-using global_time=logger<logger_global_time, verbatim_formatter, cout_sink_provider>;
-using local_time=logger<logger_local_time, verbatim_formatter, cout_sink_provider>;
+using info=logger<logger_info, cadmium::logger::formatter<float>, cout_sink_provider>;
```

Comment:
Whats the meaning of float as parameter of the formatter?

Reply:
The float as parameter is the model TIME type. It is needed in order to allow some formatter methods to take the time as parameter too.

For example, when logging the global time, then the formatter needs to take the global time to format it and then output the formatted string of the global time event.

If you check the old code, you will notice that this parameter it was already there in cadmium::logger::coordinator_formatter and camium::logger::simulator_formatter. I have removed these modules from being directly used in the simulator and coordinator now they comes in the logger formatter type, thus they have the model TIME parameter.